internal/postgres/application_service.go

```diff
@@ -290,3 +290,51 @@ func (s *ApplicationService) GetByUserID(uid int) (*models.Application, error) {
 	err = tx.Commit()
 	return dba.toModel(), err
 }
+
+func (s *ApplicationService) GetUserCount() int {
+	tx, err := s.DB.Begin()
+	if err != nil {
+		return -1
+	}
+
+	rows, err := tx.Query("SELECT COUNT(*) FROM users")
+
+	if err != nil {
+		return -1
+	}
+	defer rows.Close()
+
+	var count int
+
+	for rows.Next() {
+		if err := rows.Scan(&count); err != nil {
+			return -1
+		}
+
+	}
+	return count
+}
+
+func (s *ApplicationService) GetApplicationCount() int {
```

Comment:
Same comments as `GetUserCount` but this one should stay in `application_service.go`.